<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordsWindow</class>
 <widget class="QDialog" name="RecordsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table of records</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/icons/icon.ico</normaloff>:/resources/icons/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Level</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Saves</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="m_easy">
         <property name="text">
          <string>Easy:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_easy_name">
         <property name="text">
          <string>[Name]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_easy_time">
         <property name="text">
          <string>[Time]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_easy_saves">
         <property name="text">
          <string>[Saves]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Medium:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_medium_name">
         <property name="text">
          <string>[Name]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_medium_time">
         <property name="text">
          <string>[Time]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_medium_saves">
         <property name="text">
          <string>[Saves]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Hard:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_hard_name">
         <property name="text">
          <string>[Name]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_hard_time">
         <property name="text">
          <string>[Time]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_hard_saves">
         <property name="text">
          <string>[Saves]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Hardcore:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_hardcore_name">
         <property name="text">
          <string>[Name]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_hardcore_time">
         <property name="text">
          <string>[Time]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_hardcore_saves">
         <property name="text">
          <string>[Saves]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_ok_btn">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_ok_btn</sender>
   <signal>clicked()</signal>
   <receiver>RecordsWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
